docs/providers/aws/events/websocket.md

```diff
@@ -296,3 +296,17 @@ provider:
     websocket:
       fullExecutionData: false
 ```
+
+## Tags
+
+When using Websocket API, it is possible to tag the corresponding API Gateway resources. By setting `provider.websockets.useProviderTags` to `true`, all tags defined on `provider.tags` will be applied to API Gateway and API Gateway Stage.
+
+```yaml
+provider:
+  tags:
+    project: myProject
+  websockets:
```

Comment:
Let's use `provider.websocket` instead of `provider.websockets` to be consistent with the event name